cypress/e2e/11-support-manage-facility.cy.js

```diff
@@ -1,29 +1,31 @@
-import {loginHooks, testNumber} from "../support/e2e";
-import {addMockFacility, whoAmI} from "../utils/testing-data-utils";
-import {graphqlURL} from "../utils/request-utils";
-import {aliasGraphqlOperations} from "../utils/graphql-test-utils";
+import { loginHooks, testNumber } from "../support/e2e";
+import { addMockFacility, whoAmI } from "../utils/testing-data-utils";
+import { graphqlURL } from "../utils/request-utils";
+import { aliasGraphqlOperations } from "../utils/graphql-test-utils";
 
 loginHooks();
 describe("Support admin: manage facility", () => {
-  let organizationId="";
+  let organizationId = "";
   let facilityId = "";
   let facilityCreated = {
-    id:"",
-    name:`RainbowCenter-${testNumber()}`,
-  }
+    id: "",
+    name: `RainbowCenter-${testNumber()}`
+  };
 
   before(() => {
-    addMockFacility(facilityCreated.name).then(response=>{
+    addMockFacility(facilityCreated.name).then(response => {
       facilityCreated.id = response.body.data.addFacility.id;
     });
 
     whoAmI().then((res) => {
       organizationId = res.body.data.whoami.organization.id;
       facilityId = res.body.data.whoami.organization.facilities[0].id;
     });
+  });
 
+  beforeEach(() => {
```

Comment:
moving this hook into a before each as recommended [here](https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Aliases-are-reset-before-each-test)